src/main/java/mate/academy/bookstore/repository/user/UserRepository.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.bookstore.repository.user;
+
+import jakarta.validation.constraints.Email;
+import java.util.Optional;
+import mate.academy.bookstore.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByEmail(@Email String email);
```

Comment:
```suggestion
    Optional<User> findByEmail(String email);
```
you already validated request dto